src/app/saga.ts

```diff
@@ -93,14 +96,22 @@ const DO_NOT_LOCK_PERIOD = 30000 // 30 sec
 // Work that's done before other sagas are initalized
 // Be mindful to not put long blocking tasks here
 export function* appInit() {
-  yield call(initializeSentry)
+  SentryTransactionHub.startTransaction(SentryTransaction.app_tooling_initialised)
+  const t0 = Date.now()
```

Comment:
nit: it's better to use a [monotonic clock](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now#performance.now_vs._date.now), which don't suffer from possible time adjustments.
```suggestion
  const t0 = performance.now()
```